src/compas/files/obj.py

```diff
@@ -1,15 +1,18 @@
 from __future__ import print_function
 from __future__ import absolute_import
 from __future__ import division
-
 from collections import OrderedDict
-
 try:
     from urllib.request import urlopen
 except ImportError:
     from urllib2 import urlopen
+try:
+    from pathlib import Path
+except ImportError:
+    from pathlib2 import Path
```

Comment:
is this necessary?
should path related functionality not be handled by the base reader?

Reply:
True. Fixed :).